charts/sysdig/templates/clusterrole.yaml

```diff
@@ -105,4 +105,23 @@ rules:
     verbs:
       - "use"
 {{- end }}
+{{- if .Values.cspmCollector.deploy  }}
```

Comment:
This might make sense to put as a separate clusterrole + clusterrolebinding to the service account, since this is only for the cspm collector, not the agent

Reply:
Thanks! I added a clusterole and a clusterrolebinding for the cspm collector.